docs/source/core/partition.rst

```diff
@@ -888,7 +889,8 @@ documentation on how to run the API as a container locally.
   filename = "example-docs/eml/fake-email.eml"
 
   elements = partition_via_api(
-    filename=filename, api_url="http://localhost:5000/general/v0/general"
+    filename=filename,
+    api_url="http://<<REPLACE WITH YOUR API KEY>>/general/v0/general"
```

Comment:
same fix needed here, this is URL and not key, and it would be nice to include api_key in this example snippet

Reply:
Good point. I've added the `api_key` and used `https` in the `api_url`.